postpone/postpone.c

```diff
@@ -761,27 +761,29 @@ int mutt_get_postponed(struct Mailbox *m_cur, struct Email *hdr,
        * see http://dev.mutt.org/trac/ticket/3653 */
       rc |= SEND_POSTPONED_FCC;
     }
+    /* Note: `plen` is only assigned once due to the short-circuit behaviour of
+     * `||` */
     else if (((WithCrypto & APPLICATION_PGP) != 0) &&
              /* this is generated by old neomutt versions */
-             (mutt_str_startswith(np->data, "Pgp:") ||
+             ((plen = mutt_str_startswith(np->data, "Pgp:")) ||
```

Comment:
In the original code, we're doing case-**sensitive** comparisons.
(these are headers that NeoMutt has written, rather than the user)
I think we should be forgiving (case-insensitive), like we are in `mutt_edit_headers()`.

Reply:
I noticed that, too, but didn't want change it.  Will be rectified in the next version.